shenyu-bootstrap/pom.xml

```diff
@@ -437,6 +437,13 @@
                 </exclusion>
             </exclusions>
         </dependency>
+
+        <!-- apache shenyu data sync start use nacos-->
+        <dependency>
+            <groupId>org.apache.shenyu</groupId>
+            <artifactId>shenyu-spring-boot-starter-sync-data-nacos</artifactId>
+            <version>${project.version}</version>
```

Comment:
why add this

Reply:
because use nacos sync data，please see #3512

Reply:
I will close this pr.